src/libs/api.ts

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+const api = axios.create();
```

Comment:
I see the intention, but I do not foresee any reason to ever not use axios, and if that day does come, this change can be tackled then. Let's not prematurely optimise things, it's not to scope of the repo, the specific http client used will never be important